### PR TITLE
Ownwall posts

### PR DESCRIPTION
**Pull request by robertwoolley99**
Walls:

Can read posts on own and others walls
Can write posts on own and others walls
Posts on walls also appear in news feed.
Time formatting cleaned up on posts.
Post author now appears.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -4,8 +4,13 @@ class PostsController < ApplicationController
 
   def create
     @post = current_user.posts.create(post_params)
-    # redirect_to posts_url
-    redirect_to posts_url
+
+    if (@post.wall_id == 0)
+      redirect_to posts_url
+    else
+      @user_url = '/users/' + @post.wall_id.to_s
```
Comment (KOlofinmoyin) [inline app/controllers/posts_controller.rb]: .. hey Rob, dyou mean to use an '_assignment operator_' = or an 'equality operator' == here?

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,21 @@
...
+<%= form_for @post do |form| %>
+  <%= form.label :message %>
+  <%= form.text_area :message %>
+<%= form.hidden_field :wall_id, :value => @user.id %>
+  <%= form.submit "Submit" %>
+<% end %>
+
+
+
+ <h1>  users own  wall </h1>
```
Comment (KOlofinmoyin) [inline app/views/users/show.html.erb]: ... could we maybe put the User's name here e.g. _Katy's own wall_ or _Katy's wall_; in-lieu of 'users own wall'?

**Review by KOlofinmoyin — CHANGES_REQUESTED** [review]
please see the comments

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (robertwoolley99) [reply app/controllers/posts_controller.rb]: We mean assignment operator.
If post.wall_id == 0 # this is for newsfeed posts - where the wall number is 0
else
#set the url to be the wall owner's url

File: app/views/users/show.html.erb (hunk 2 above)
Comment (robertwoolley99) [reply app/views/users/show.html.erb]: Added. New commit sent.

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (KOlofinmoyin) [reply app/controllers/posts_controller.rb]: thanks Rob

**Review by KOlofinmoyin — APPROVED** [review]
:shipit: 👍

**Pull request merged**